docker-compose.env

```diff
@@ -1,12 +1,15 @@
-
-REDACT_PIPELINE_IMAGE="docker.brighter.ai/redact:v3.3.1-237"
-REDACT_INFER_IMAGE="docker.brighter.ai/redact-gpu:0.3.0"
+REDACT_IMAGE="brighter.azurecr.io/redact:v4.4.0-252"
```

Comment:
wrong repo should all come from docker.brighter.ai

Reply:
this is currently our internal repo

Reply:
done